appengine-java8/tasks/src/main/java/com/example/task/CreateTask.java

```diff
@@ -121,22 +121,36 @@ public static void main(String... args) throws Exception {
     String payload = params.getOptionValue(PAYLOAD_OPTION.getOpt(), "default payload");
 
     // [START cloud_tasks_appengine_create_task]
+    // Instantiates a client.
     try (CloudTasksClient client = CloudTasksClient.create()) {
+
+      // TODO(developer): Uncomment these lines and replace with your values.
+      // String project = "my-project-id";
```

Comment:
Vars should be `projectId` and `queueName` to match below if this is kept as is. But uncommenting these lines would be a compile error because these fields are all declared up above. In addition, they are all pulled from CLI opts so the user is presumably already supplying them if they are running the CLI, unless this comment is more aimed at docs readers than people running the sample.

Reply:
Trying to comprise between the code snippet for the docs and running the sample with the CLI. Changing the comment to "Variables provided by the CLI."